src/ghost-content/index.js

```diff
@@ -3,8 +3,11 @@ import fs from 'fs'
 import Promise from 'bluebird'
 import glob from 'glob'
 import uuid from 'uuid'
+
 import get from 'lodash/get'
 import partition from 'lodash/partition'
+import mapValues from 'lodash/mapValues'
+import uniq from 'lodash/uniq'
```

Comment:
I'd still vote for importing with destructuring. Not sure we should optimize code for bundlers (which may become unnecessary in future versions) rather then keeping readability.

Reply:
Agreed, I personally prefer destructing or importing lodash as `_` directly. But that's a very minor decision, I don't think we should enforce a specific rule for as it doesn't impact DX in any way

Reply:
Well, I like FP, point-free style where possible and this is the closest thing. I think its readability is debatable. I've seen this style in projects by well-respected authors.

Reply:
Oh guys I have some good news for you :)
https://www.npmjs.com/package/lodash-es is properly exported with ES6 exports so if we switch to it we can safely used "destructuring" imports
I will eventually do this refactoring some day, stay tuned :)